src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/EmploymentCheck/RefreshEmploymentChecksCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿using SFA.DAS.EmployerIncentives.Abstractions.Commands;
+using SFA.DAS.EmployerIncentives.Abstractions.Logging;
+
+namespace SFA.DAS.EmployerIncentives.Commands.ApprenticeshipIncentive.EmploymentCheck
+{
+    public class RefreshEmploymentChecksCommand : DomainCommand, ILogWriter
```

Comment:
Implement the IPeriodEndIncompatible to delay or prevent the command running.  You may need to improve the interface if the requirement is to abandon the command from running rather than just delay it.